ssg/templates.py

```diff
@@ -286,7 +286,7 @@ def sebool(data, lang):
     return data
 
 
-@template(["ansible", "bash", "oval", "puppet"])
+@template(["ansible", "bash", "oval", "puppet", "ignition", "kubernetes"])
```

Comment:
This needs to be added to the documentation in docs/manual/developer_guide.adoc line 1622.

Reply:
Done